packages/ckeditor5-core/src/editor/utils/dataapimixin.js

```diff
@@ -71,7 +71,8 @@ export default DataApiMixin;
  * the right format for you.
  *
  * @method #getData
- * @param {Object} [options]
+ * @param {Object} [options] Additional configuration for the retrieved data. By default two optional
```

Comment:
How about ...

```
Additional configuration for the retrieved data. Editor features may introduce more configuration options that can be set through this parameter.
```

Reply:
I think I'd leave `DataController` description as it is now. `editor.getData()` is a very top-level method, its description needs to be as simple as it can.